fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1060,20 +1060,31 @@ private void computeFragmentHosts() throws Exception {
                     TNetworkAddress key = tNetworkAddressMapEntry.getKey();
                     Map<Integer, List<TScanRangeParams>> value = tNetworkAddressMapEntry.getValue();
 
-                    for (Integer planNodeId : value.keySet()) {
-                        List<TScanRangeParams> perNodeScanRanges = value.get(planNodeId);
-                        int expectedInstanceNum = 1;
-                        if (parallelExecInstanceNum > 1) {
-                            //the scan instance num should not larger than the tablets num
-                            expectedInstanceNum = Math.min(perNodeScanRanges.size(), parallelExecInstanceNum);
+                    boolean isReplicated = isRelicatedFragment(fragment.getPlanRoot());
```

Comment:
Whether replication should take precedence over Bucketshuffle and Colocate

Reply:
No, it needn't